homeassistant/components/homee/strings.json

```
@@ -110,6 +110,16 @@
         "name": "Wake-up interval"
       }
     },
+    "select": {
+      "repeater_mode": {
+        "name": "EnOcean repeater mode",
```

```suggestion
        "name": "Repeater mode",
```
Do we need the model name as part of the entity name? The entities will already be part of a device.